plc4j/drivers/s7/pom.xml

```diff
@@ -118,6 +118,12 @@
       <version>1.12</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
I think this could eventually cause problems. By setting the scope to provided, it is not shipped with the application. So as soon as you run it in a normal environment without osgi, it could cause CNFEs when loading the class which is annotated with "@Component". Will test this.

Reply:
@chrisdutz I see your point BUT the annotation is not kept in the byte code, its scope is source only. So it will no longer be available when shipped.
(The plugin creates a xml file which is shipped with the code under OSGI-INF/...)